dsaps/cli.py

```diff
@@ -15,7 +20,13 @@
 @click.pass_context
 def main(ctx, url, email, password):
     ctx.obj = {}
-    print('Application start')
+    timestamp = datetime.datetime.now().strftime('%Y-%m-%d %H.%M.%S')
+    log_suffix = f'{timestamp}.log'
+    logging.basicConfig(format="%(message)s",
+                        handlers=[logging.FileHandler(f'log-{log_suffix}',
```

Comment:
I think the best thing to do here if writing log files would be to create a `logs` directory as part of the repo and write to that. Otherwise, your top level in the project ends up full of log files.